ckanext/data_qld/pylons_plugin.py

```diff
@@ -22,7 +22,11 @@ def before_map(self, m):
         m.connect(
             'data_qld.show_schema',
             '/dataset/{dataset_id}/resource/{resource_id}/%s/show/' % constants.SCHEMA_MAIN_PATH,
-            controller=controller, action='show_schema', conditions=dict(method=['GET']))
+            controller=controller, action='show_resource_schema', conditions=dict(method=['GET']))
+        m.connect(
+            'data_qld.show_schema',
+            '/dataset/{dataset_id}/%s/show/' % constants.SCHEMA_MAIN_PATH,
```

Comment:
It would not. The proper way of handling it is to accept an extra `package_type` parameter in the view function, and turn `/dataset/` into `/{package_type}/`

Reply:
Will be fixed and pushed by Alex shortly.